### ISSUE TITLE
Add Compatibility For Windows

### ISSUE THREAD

**Issue by thakursc1** [CLARIAH/grlc#169]
I get the following error when running the app on windows.  

````
 gunicorn -c gunicorn_config.py src.server:app
Traceback (most recent call last):
  File "c:\programdata\anaconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\programdata\anaconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\ProgramData\Anaconda3\Scripts\gunicorn.exe\__main__.py", line 5, in <module>
  File "c:\programdata\anaconda3\lib\site-packages\gunicorn\app\wsgiapp.py", line 9, in <module>
    from gunicorn.app.base import Application
  File "c:\programdata\anaconda3\lib\site-packages\gunicorn\app\base.py", line 12, in <module>
    from gunicorn import util
  File "c:\programdata\anaconda3\lib\site-packages\gunicorn\util.py", line 9, in <module>
    import fcntl
ModuleNotFoundError: No module named 'fcntl'

**Comment by c-martinez** [comment]
Gunicorn does not support windows -- and grlc runs on gunicorn.

Looking at [this issue](https://github.com/benoitc/gunicorn/issues/524) it looks like they are planning on adding windows support (and maybe it [already works on python3.7](https://ci.appveyor.com/project/benoitc/gunicorn/build/job/ysfbmipm5yvix4o2)?)

@thakursc1 can I ask what version of python you are using?

**Comment by thakursc1** [comment]
3.6

**Comment by c-martinez** [comment]
Ok -- do you have any way of running 3.7 ? May be worth trying , although I have no idea if it would work.

**Comment by thakursc1** [comment]
Doesn't help @c-martinez

**Comment by c-martinez** [comment]
Is it possible to run it installed via pip:
```
pip install grlc
grlc-server
```

or running the docker image (although I think docker and windows sometimes causes issues too):
```
git clone https://github.com/CLARIAH/grlc
cd grlc
docker pull clariah/grlc
docker-compose -f docker-compose.default.yml up
```

Sorry for not being able to provide concrete help -- I don't have a windows machine, so I am shooting in the dark a bit.

**Comment by c-martinez** [comment]
Hi @thakursc1 -- I'm not sure if you managed to solve your issue, but I just had an thought: it might be possible to use [waitress](https://pylons.readthedocs.io/projects/waitress/en/latest/) instead of `gunicorn` to run grlc on windows.

If it is still interesting, perhaps we could give this a go? I am happy to update instructions / make changes to support `waitress`, but I would need your help to test on windows, if possible. Would that help?

**Comment by Sweenpet** [comment]
@thakursc1 , have you tried running it under WSL?

**Comment by c-martinez** [comment]
Closed by #175 -- windows support is now provided!
